### DISCUSSION TITLE
https://finicky-kickstart.now.sh config helper doesn't work...

### DISCUSSION THREAD

**Discussion by MikeGit06** [johnste/finicky#256] [General]
Hello,

I tried using the following url : https://finicky-kickstart.now.sh
to edit Finicky's basic configuration but the generated code doesn't work...
Links keep opening with Safari.
Too bad, because it would be a practical way to edit its configuration for an amateur user.

This code works for me :

![Screen2](https://user-images.githubusercontent.com/102796453/161238085-43b78b96-a462-4211-a7e0-b7ade8d11f1f.png)

But it's not the same as on the https://finicky-kickstart.now.sh website.

![ScreenShot](https://user-images.githubusercontent.com/102796453/161236666-d1d778ef-94dd-43a7-885b-785340d8ce9e.png)

It would be cool if that worked.

Anyway, thank you very much John for this cool app.

Mike

**Comment by johnste** [comment]
I will try to have a look, thanks for reporting!
